uint32 matrix_id
float64 force
float64 contact_area
bool in_contact
float64 x_center
float64 y_center